gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/assembly/plugin-assembly-distribution.xml

```diff
@@ -169,6 +169,7 @@
                 <exclude>io.gravitee.policy:gravitee-gateway-services-ratelimit:zip</exclude>
                 <exclude>io.gravitee.reporter:*:zip</exclude>
                 <exclude>io.gravitee.tracer:*:zip</exclude>
+                <exclude>com.graviteesource.service:gravitee-service-secrets:zip</exclude>
```

Comment:
excluding a zip file here is not enough. Unfortunately, we still have 2 ways to build our release and our docker images.
The first one uses maven assembly plugin for our dev docker images.
The second one uses a script for our prod images.

You also need to change this script (https://github.com/gravitee-io/gravitee-api-management/blob/a7dd5bf221cfadad15a2aa1eb6c44f4829f93a2b/release/ci-steps/package-bundles.mjs#L156-L169) to add your exclusion